src/pages/latest/delta-update.mdx

```diff
@@ -410,24 +410,31 @@ Here is a detailed description of the `merge` programmatic operation.
 
 ### Automatic schema evolution
 
-By default, `updateAll` and `insertAll` assign all the columns in the target Delta table with columns of the same name from the source dataset. Any columns in the source dataset that don't match columns in the target table are ignored. However, in some use cases, it is desirable to automatically add source columns to the target Delta table. To automatically update the table schema during a `merge` operation with `updateAll` and `insertAll` (at least one of them), you can set the Spark session configuration `spark.databricks.delta.schema.autoMerge.enabled` to `true` before running the `merge` operation.
+Schema evolution allows users to resolve schema mismatches between the target and source table in merge. It handles the following two cases:
 
-<Info title="Note" level="info">
+#. A column in the source table is not present in the target table. The new column is added to the target schema, and its values are inserted or updated using the source values.
```

Comment:
The list numbers using `#.` is not rendering properly locally for me but I see other places using this so.. it should work in the actual doc?